airbyte-integrations/connectors/source-postgres/src/main/resources/spec.json

```diff
@@ -104,21 +104,21 @@
               "plugin": {
                 "type": "string",
                 "title": "Plugin",
-                "description": "A logical decoding plug-in installed on the PostgreSQL server. `pgoutput` plug-in is used by default.\nIf replication table contains a lot of big jsonb values it is recommended to use `wal2json` plug-in. For more information about `wal2json` plug-in read <a href=\"https://docs.airbyte.com/integrations/sources/postgres\">Postgres Source</a> docs.",
+                "description": "A logical decoding plug-in installed on the PostgreSQL server. `pgoutput` plug-in is used by default.\nIf replication table contains a lot of big jsonb values it is recommended to use `wal2json` plug-in. For more information about `wal2json` plug-in read <a href=\"https://docs.airbyte.com/integrations/sources/postgres#replication-plugin\">Select replication plugin</a> in documentation.",
```

Comment:
nitpick: I would prefer to just say "read about XYZ" than "read about XYZ in documentation" for all of these links - it just feels a little redundant

Reply:
done